ros2_control_demo_hardware/include/ros2_control_demo_hardware/rrbot_system_multi_interface.hpp

```diff
@@ -74,9 +75,13 @@ class RRBotSystemMultiInterfaceHardware : public hardware_interface::SystemInter
   std::vector<double> hw_commands_positions_;
   std::vector<double> hw_commands_velocities_;
   std::vector<double> hw_commands_accelerations_;
-  std::vector<double> hw_positions_;
-  std::vector<double> hw_velocities_;
-  std::vector<double> hw_accelerations_;
+  std::vector<double> hw_states_positions_;
+  std::vector<double> hw_states_velocities_;
+  std::vector<double> hw_states_accelerations_;
+
+  // Store time between update loops
+  std::chrono::time_point<std::chrono::system_clock> last_timestamp_;
```

Comment:
does this not get into trouble with ROS time vs wall time, etc ? wouldn't it be better to use rclcpp::Time instead?
...and below for `duration`?